pynamodb/migration.py

```diff
@@ -79,15 +83,21 @@ def migrate_boolean_attributes(model_class,
                                             meant to trigger the code path in boto, to allow us to unit test that
                                             we are jumping through appropriate hoops handling the resulting
                                             failure and distinguishing it from other failures.
+    :param page_size: Passed along to the underlying 'page_size'. Page size of the scan to DynamoDB.
+    :param limit: Passed along to the underlying 'limit'. Used to limit the number of results returned.
+    :param back_off: Number of seconds to sleep when exceeding capacity.
 
     :return: (number_of_items_in_need_of_update, number_of_them_that_failed_due_to_conditional_update)
     """
     log.info('migrating items; no progress will be reported until completed; this may take a while')
     num_items_with_actions = 0
     num_update_failures = 0
+    items_processed = 0
 
     for item in model_class.rate_limited_scan(_build_lba_filter_condition(attribute_names),
                                               read_capacity_to_consume_per_second=read_capacity_to_consume_per_second,
+                                              page_size=page_size,
+                                              limit=limit,
                                               allow_rate_limited_scan_without_consumed_capacity=allow_rate_limited_scan_without_consumed_capacity):
         actions = []
         condition = None
```

Comment:
something super irrelevant that I wanna bring up here, correct me if you disagree. 
it feels the section between 103 to 139 can use some help of a helper function. 
separating the logic into smaller funcs always improve the readability.
since we are touching this piece of code anyway, do we want to spend a little bit more time to refactor this really quick ? 
it is really hard to read through and understand what is going on.

Reply:
I do agree that the code could use some comments, but I find the two parts of the main function : 1-build action list & 2- if actions, do them, are pretty minimal. Also this is -kinda of tmp- migration code, that we need to run soon on production machines, we could plan this for later maybe..

Reply:
well, since this is tmp and will be deprecated soon ( I assume ), it is fine to let it be like this.